Sources/CapriccioLib/SwiftTestsFilesWriter.swift

```diff
@@ -38,14 +39,17 @@ public final class SwiftTestsFilesWriter {
         write(code: singleFileCode, toFile: singleFilePath)
     }
     
-    private func writeFeatureFiles(fromFeaturesCode featuresCode: [String], features: [Feature], folderPath: String) {
-        for i in 0..<features.count {
+    private func writeFeatureFiles(fromFeaturesCode featuresCode: [String], metadatas: [Metadata], folderPath: String) {
+        for i in 0..<metadatas.count {
```

Comment:
Nitpick: you can use `for (index, metadata) in metadatas.enumerated()`, then you can remove `let metadata = metadatas[i]` (I know was there from before, but I thought was still a good idea highlight it, because I think with that change the code would be better :) )
But is not a blocker